File: Baekjoon/Swift/BOJ_PS/BOJ_PS/Implementation/Q10988-checkPalindrome.swift
//
//  Q10988-checkPalindrome.swift
//  BOJ_PS
//
//  Created by thoonk on 2022/08/10.
//  팰린드롬인지 확인하기 / 구현

let input = Array(readLine()!.map { String($0) })
var isPalindrome = true

for i in 0..<input.count/2 {
    if input[i] != input[input.count - i - 1] {
        isPalindrome = false
    }
}

print(isPalindrome ? 1 : 0)

File: Baekjoon/Python/BOJ_PS/Implementation/Q10988-checkPalindrome.py
# 팰린드롬인지 확인하기 / 구현
input = list(input())
isPalindrome = True
for i in range(len(input) // 2):
    if input[i] != input[len(input) - i - 1]:
        isPalindrome = False

print(1 if isPalindrome else 0)